ga4gh/application.wsgi

```diff
@@ -0,0 +1,3 @@
+from ga4gh.frontend import app as application
```

Comment:
Absolutely. I'll leave that question for another day :) 
Moved the file and copied it in place with the pre-build hook. Diff [here](https://github.com/UMMS-Biocore/server/commit/5785743e401f028cbf323fe0186bcc04d031a41d), squashed into this PR